### ISSUE TITLE
questions about Embedding and WordContextProduct using Convolution1D  doing NLP

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#629]
Does 'Embedding' use the skipgram method by Mikolov? According to several rows codes, I can't understand how to product the word vectors. What about 'WordContextProduct' in embedding.py?
Many thanks! Please reply as soon as possible!

**Comment by fchollet** [comment]
> Does 'Embedding' use the skipgram method by Mikolov?

No. It turns sequences of integer indices into dense vectors. This mapping is learned via gradient descent.

> What about 'WordContextProduct' in embedding.py?

Take a look at https://github.com/fchollet/keras/blob/master/examples/skipgram_word_embeddings.py
